engine/lib/luajit-ffi-gen/src/impl_item/parse.rs

```diff
@@ -109,12 +117,21 @@ fn parse_method_attrs(attrs: &mut Vec<Attribute>) -> Result<(Vec<String>, BindAr
     }
 }
 
-fn parse_params<'a>(
+fn parse_params<'a, 'b>(
```

Comment:
Do we really need a second lifetime? Should not params and generics live in the same scope?

Reply:
I think VSCode added this by itself, I don't remember adding this...